libs/SalesforceHybrid/src/com/salesforce/androidsdk/phonegap/ui/SalesforceDroidGapActivity.java

```diff
@@ -130,48 +133,37 @@ protected CordovaWebViewEngine makeWebViewEngine() {
     @Override
     public void onResume() {
         super.onResume();
-        if (passcodeManager.onResume(this)) {
+        delegate.onResume(false);
+        // will call this.onResume(RestClient client) with a null client
+    }
 
-            // Get client (if already logged in)
-            try {
-                client = clientManager.peekRestClient();
-            } catch (AccountInfoNotFoundException e) {
-                client = null;
-            }
+    @Override
+    public void onResume(RestClient _) {
```

Comment:
Is `_` for parameter in `Java 8`?

Reply:
It works with swift, but the compiler is warning that it might no be supported in the future, so I changed it to something else.